vendor/engines/calendar/lib/weby/components/event_list/event_list_component.rb

```diff
@@ -13,8 +13,7 @@ def get_events(site, page_param)
 
   def events(site, page_param)
     direction = 'desc'
-    events = Calendar::Event.where(site_id: site.id).includes(:image).upcoming
-      .order("begin_at asc, id asc").page(page_param).per(quant)
+    site.events.includes(:image).upcoming.order("begin_at asc, id asc").page(page_param).per(quant)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.